src/snowflake/snowpark/modin/plugin/_internal/apply_utils.py

```diff
@@ -222,8 +222,11 @@ def end_partition(self, df):  # type: ignore[no-untyped-def] # pragma: no cover
 
             result["value"] = (
                 result["value"]
-                .apply(handle_missing_value_in_variant)
-                .apply(convert_numpy_int_result_to_int)
+                .apply(
```

Comment:
I agree with @sfc-gh-mvashishtha those changes should not have a significant perf impact.